lib/foreigner/schema_dumper.rb

```diff
@@ -38,6 +38,14 @@ def remove_prefix_and_suffix(table_name)
         end
       end
       module_function :remove_prefix_and_suffix
+
+      def maybe_prepend_public(table_name)
```

Comment:
You may wish to consider using `Apartment.default_schema` as opposed to `public` in this method, as [this is configurable](https://github.com/influitive/apartment#providing-a-different-default_schema).